pandas/core/nanops.py

```diff
@@ -1067,22 +1067,14 @@ def reduction(
         axis: AxisInt | None = None,
         skipna: bool = True,
         mask: npt.NDArray[np.bool_] | None = None,
-    ) -> Dtype:
```

Comment:
Can we keep this type annotation?

Reply:
This annotation was wrong and this doesn't return a `Dtype` but the the result of a min/max of a DataFrame/Series, i.e. typically a scalar or a ndarray (but the min/max of a Series can in principle be anything....)

IDK if we return type `Scalar` as annotation for Series?

Reply:
Yeah not either, but since this is an inner function the typing here probably doesn't matter as much